<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>integral</class>
 <widget class="QDialog" name="integral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Integral</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignCenter';
}</string>
   </property>
   <property name="text">
    <string>Решение определенного интеграла</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>190</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Введите интегрируемую функцию и пределы игтегрирования*:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>290</y>
     <width>223</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Cute-Ball-Help-icon.png</normaloff>:/images/Cute-Ball-Help-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>180</y>
     <width>41</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/images/638px-WPint.svg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*пределы интегрирования могут принимать только численные значения</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
      </property>
      <property name="text">
       <string>b =</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_b">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignCenter';
  border: 1px solid gray;
}

background-color : white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
      </property>
      <property name="text">
       <string>a =</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_a">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignCenter';
  border: 1px solid gray;
}

background-color : white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>741</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
}</string>
      </property>
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_res">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignLeft';
	border : 1px solid gray;
}

background-color : white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_clear_res">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
      </property>
      <property name="text">
       <string>Очистить строку результата</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
      </property>
      <property name="text">
       <string>Очистить строку ввода</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_clear_limits">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
      </property>
      <property name="text">
       <string>Очистить поля границ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>200</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="write_integral">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_dx">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignLeft';
}

background-color : white;</string>
      </property>
      <property name="text">
       <string>dx</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_a</tabstop>
  <tabstop>lineEdit_b</tabstop>
  <tabstop>write_integral</tabstop>
  <tabstop>pushButton_calculate</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>pushButton_clear_limits</tabstop>
  <tabstop>pushButton_clear_res</tabstop>
  <tabstop>pushButton_help</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>integral</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>916</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>876</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
